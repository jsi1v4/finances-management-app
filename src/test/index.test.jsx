import React from 'react';
import { render } from '@testing-library/react';

import Home from '../pages/index';

describe('Index component', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Home />, {});
    expect(asFragment()).toMatchSnapshot();
  });

  it('should render text component', async () => {
    const { getByText } = render(<Home />, {});
    expect(getByText('Hello World!!!').innerHTML).toBeDefined();
  });
});
