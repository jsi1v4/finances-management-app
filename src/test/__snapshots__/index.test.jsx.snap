// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index component matches snapshot 1`] = `
<DocumentFragment>
  <h1
    data-testid="hello"
  >
    Hello World!!!
  </h1>
</DocumentFragment>
`;
